crates/common/types/genesis.rs

```diff
@@ -200,6 +200,19 @@ pub struct ChainConfig {
     pub deposit_contract_address: Address,
 }
 
+lazy_static::lazy_static! {
+    pub static ref NETWORK_NAMES: HashMap<u64, &'static str> = {
+        HashMap::from([
+            (1, "mainnet"),
+            (11155111, "sepolia"),
+            (17000, "holesky"),
+            (560048, "hoodi"),
+            (9, "L1 local dev network"),
+            (65536999, "L2 local dev network"),
```

Comment:
Nit: we can shorten it to "devnet"

```suggestion
            (9, "L1 local devnet"),
            (65536999, "L2 local devnet"),
```